esphome/components/micronova/switch/__init__.py

```diff
@@ -0,0 +1,53 @@
+import esphome.codegen as cg
+import esphome.config_validation as cv
+from esphome.components import switch
+
+from .. import (
+    MicroNova,
+    MicroNovaFunctions,
+    CONF_MICRONOVA_ID,
+    CONF_MEMORY_LOCATION,
+    CONF_MEMORY_ADDRESS,
+    MICRONOVA_LISTENER_SCHEMA,
+    micronova_ns,
+)
+
+CONF_STOVE_SWITCH = "stove_switch"
```

Comment:
This doesnt need a suffix, its already under `switch`
```suggestion
CONF_STOVE = "stove"
```